site/docs/tools/producer_tools.md

```diff
@@ -0,0 +1,7 @@
+# Producer Tools
```

Comment:
Are these linked into the main site automatically somehow?

Reply:
Yes, mkdocs should handle everything.